app/models/jurisdiction.rb

```diff
@@ -20,4 +20,8 @@ def display_full
     result.concat(" (#{abbr})") unless abbr.blank?
     result
   end
+
+  def self.pluck_h(keys)
```

Comment:
Why the name **pluck_h**?
Maybe it should have a name that describe what is meant to do?